pkgs/tools/security/ssh-mitm/default.nix

```diff
@@ -1,55 +1,43 @@
-{ lib
-, fetchFromGitHub
-, python3
+{ lib,
+fetchFromGitHub,
+python3,
+stdenv
 }:
 
-let
-  py = python3.override {
-    packageOverrides = self: super: {
-      paramiko = super.paramiko.overridePythonAttrs (oldAttrs: rec {
-        version = "3.1.0";
-        src = oldAttrs.src.override {
-          inherit version;
-          hash = "sha256-aVD6ymgZrNMhnUrmlKI8eofuONCE9wwXJLDA27i3V2k=";
-        };
-        patches = [ ];
-        propagatedBuildInputs = oldAttrs.propagatedBuildInputs ++ [ python3.pkgs.icecream ];
-      });
-    };
-  };
-in
-with py.pkgs;
-
-buildPythonApplication rec {
+python3.pkgs.buildPythonApplication rec {
   pname = "ssh-mitm";
-  version = "3.0.2";
+  version = "4.1.1";
   format = "setuptools";
 
   src = fetchFromGitHub {
     owner = pname;
     repo = pname;
     rev = "refs/tags/${version}";
     hash = "sha256-koV7g6ZmrrXk60rrDP8BwrDZk3shiyJigQgNcb4BASE=";
+    # Correct hash does not seems to build ??
+    #hash = "sha256-Uf1B7oEZyNWj4TjrLvEfFdxsvsGeMLXFsSdxGLUV4ZU=";
   };
 
-  propagatedBuildInputs = [
+  propagatedBuildInputs = with python3.pkgs; [
     argcomplete
     colored
     packaging
     paramiko
     pytz
     pyyaml
     rich
+    tkinter
     setuptools
     sshpubkeys
-  ];
+  ] ++ lib.optionals stdenv.darwin [ setuptools ];
```

Comment:
This doesn't work, should probably be stdenv.isDarwin

Reply:
Yes sorry I just noticed the mistake, fixed it, now

Reply:
ofborg will probably evaluate too but I don't think it does for darwin, I may be wrong however.

Reply:
Good now ? @marcusramberg